### PR TITLE
Adding new Trusted Profile Analyzer QSG

### PR DESCRIPTION
**Pull request by ritz303**
This PR adds two new Quick Starts guides for the Service Preview release of the Trusted Profile Analyzer (TPA):

- tpa-searching-for-vulnerability-information
- tpa-scanning-an-sbom-file

These are to be added to the Learning Resources section under Application and Data Services.

Fixes: Jira [TC-332](https://issues.redhat.com/browse/TC-332)

@Hyperkid123 @ryelo : I've opened Jira task [RHCLOUD-29959](https://issues.redhat.com/browse/RHCLOUD-29959) for adding these QSG to HCC.  Please don't hesitate to yell at me if you have any questions.  Thank you for your help!

### CODE REVIEW

File: docs/quickstarts/tpa-scanning-an-sbom-file/tpa-scanning-an-sbom-file.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: tpa-scanning-an-sbom-file
+  generation: 1
+  instructional: true
+spec:
+  displayName: Scan a software bill of materials file
+  durationMinutes: 5
+  icon: >-
```
Comment (Hyperkid123) [inline docs/quickstarts/tpa-scanning-an-sbom-file/tpa-scanning-an-sbom-file.yml:10]: We are not displaying icons in the frontend. You can replace the string with `~`

File: docs/quickstarts/tpa-searching-for-vulnerability-information/tpa-searching-for-vulnerability-information.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: tpa-searching-for-vulnerability-information
+  generation: 1
+  instructional: true
+spec:
+  displayName: Search for vulnerability information
+  durationMinutes: 5
+  icon: >-
```
Comment (Hyperkid123) [inline docs/quickstarts/tpa-searching-for-vulnerability-information/tpa-searching-for-vulnerability-information.yml:10]: ^^

**Comment by Hyperkid123** [comment]
You are also missing the `color` attribute in the content that is used for the badge in the UI.

File: docs/quickstarts/tpa-scanning-an-sbom-file/tpa-scanning-an-sbom-file.yml (hunk 1 above)
Comment (ritz303) [reply docs/quickstarts/tpa-scanning-an-sbom-file/tpa-scanning-an-sbom-file.yml:10]: Hmm... I'm not sure I understand why.  When I look that the tiles (cards) on the [Learning Resources page](https://console.stage.redhat.com/application-services/learning-resources), the tiles there have icons.  Looking at the "openshift-ai-rhods-fasttrack" quick start, and it has a base64 "icon" string, and the icon displays on the tile.  I would like to keep the icon.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**